docs/src/covariance_parameters/general.md

```diff
@@ -0,0 +1,15 @@
+# General 
+
+```@meta
+CurrentModule = MarkovKernels
+```
+
+MarkovKernels.jl supports covariance matrices of the following represenration:
```

Comment:
Does this mean I can't use my own matrix type for covariances?

Reply:
No. It means you would have to implement some methods for your types containing your custom type (ideally, it would just work if it implements the "covariance interface"). 

Perhaps the following would be a better phrasing: 

> MarkovKernels.jl  currently implements functionality for covariance matrices of the following representation:

Reply:
I see. Just wanted to make sure that this doesn't imply type restrictions for all the cool methods I'd want to use with PSDMatrices.jl